.circleci/set-up-globals.sh

```diff
@@ -18,12 +18,11 @@ source $BASH_ENV
 set -ex
 
 # Update terminus temporarily.
-cd /opt/terminus
-git fetch
-git checkout 3.x
-composer install
 rm /usr/local/bin/terminus
-ln -s /opt/terminus/bin/t3 /usr/local/bin/terminus
+mkdir ~/terminus && cd ~/terminus
+curl -L https://github.com/pantheon-systems/terminus/releases/download/`curl --silent "https://api.github.com/repos/pantheon-systems/terminus/releases/latest" | perl -nle'print $& while m#"tag_name": "\K[^"]*#g'`/terminus.phar --output terminus
```

Comment:
Is it correct that there shouldn't be a space after `download/`?

Reply:
I guess so.  For readability, it seems that the nested curl statement could be on a line before this and then included in the download URL as a varible.

Reply:
I pushed this to address that: https://github.com/pantheon-systems/terminus-build-tools-plugin/pull/427/commits/1cfa0b716c897393f22c2ad910df6501f5287a43